www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/index.tsx

```diff
@@ -54,6 +54,49 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
+interface AdditionalGraphActionsProps {
+  details: ResourceDetails | undefined;
+  resource: ResourceDetails | Resource | undefined;
+  sendReloadGraphPerformance: (value: boolean) => void;
+}
+
+const AdditionalGraphActions = ({
```

Comment:
Can you extract it to the AnomalyDetection folder? And thus rename it AnomalyDetectionGraphActions